crates/humanode-runtime/src/robonode.rs

```diff
@@ -11,7 +11,12 @@ impl PublicKey {
     pub fn from_bytes(
         bytes: &[u8],
     ) -> Result<Self, robonode_crypto::ed25519_dalek::ed25519::Error> {
-        let actual_key = robonode_crypto::PublicKey::from_bytes(bytes)?;
+        if bytes.len() != 32 {
+            return Err(robonode_crypto::ed25519_dalek::ed25519::Error::new());
```

Comment:
Is it possible to add more info into error?

Reply:
Unfortunately, this is not possible with their API. I think it is intentional. We could add our own wrapper error, but what would be want to put there?

Reply:
I was asking just to get if it's possible to note that the reason is in bytes len.